src/DynamoCore/Core/PreferenceSettings.cs

```diff
@@ -102,6 +102,16 @@ public int MaxNumRecentFiles
         public List<string> BackupFiles { get; set; }
 
         /// <summary>
+        /// A list of folders containing custom nodes.
+        /// </summary>
+        public List<string> CustomNodeFolders { get; set; }
+
+        /// <summary>
+        /// A list of folders containing zero-touch nodes.
+        /// </summary>
+        public List<string> PackageFolders { get; set; } 
+
+        /// <summary>
```

Comment:
If they are IEnumerable, serialization will fail. Why should they be IEnumerable while BackupFiles is List<string>?